crates/redstone/src/core/mod.rs

```diff
@@ -3,7 +3,7 @@ pub mod processor;
 pub mod processor_result;
 
 mod aggregator;
-pub mod validator;
```

Comment:
minor: this is just mine personal preferences to avoid using `(super)` not in the tests.  I also dont like `(crate)` but that is lesser evil :P. To give a reason to that:
* It is simpler, in ideal world with only pub and non-pub qualifiers there is much less congnitive overload :P
* Again, in my personal opinion, using (super) or (crate) usually means something went wrong earlier in the design :P 

That being said, after 3-5 lines of paragraph over one super :P I would be fine with just a pub(crate) :D

Reply:
Very well presented arguments. Will fix of course. Thanks.